core/Cargo.toml

```diff
@@ -30,3 +30,6 @@ chrono = "0.4.4"
 
 grin_keychain = { path = "../keychain", version = "1.0.0" }
 grin_util = { path = "../util", version = "1.0.0" }
+
+[dev-dependencies]
```

Comment:
Anyone have any major problems with this dev-dependency? Would rather avoid but seems best way to run serialization tests.